libr/core/cmd.c

```diff
@@ -1188,6 +1188,22 @@ static int cmd_ls(void *data, const char *input) { // "ls"
 			eprintf ("Usage: less [file]\n");
 		}
 		break;
+	case 'j': { // "lj"
+		PJ * pj = pj_new();
+		pj_a (pj);
+		pj_o (pj);
+		char * res = r_syscmd_ls (arg);
```

Comment:
can return null, on sandbox mode for example, so better stop early

Reply:
Stop early, in the sense?
I did not understand that part.

Reply:
well. instead of early stop its probably better to show the actual error or just an empty listr of files.

Reply:
Get the list of files instead of getting a string and construct a proper JSON with the information of each file (size, permissions, ...)

Reply:
Use r_sys_dir(arg) and iterate over the list